composer.json

```diff
@@ -37,7 +37,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.3",
```

Comment:
Updated minimum php version to 7.3 to force to use compatible with php8 version of doctrine/dbal (2.12)

Reply:
@tolkamsoft I'd rather us only require `doctrine/dbal:2.12.x` if using PHP 8, as opposed to upping the minimum PHP version to 7.3. The October 1.1 branch supports PHP 7.2 for the moment.

Reply:
@bennothommo nope, this would not work - doctrine/dbal 2.12.0 requires php ^7.3 || ^8

Reply:
@tolkamsoft But the only reason we need `2.12.0` is for PHP 8 compatibility, so we can still use the older versions of DBAL for PHP 7.2?

Reply:
@bennothommo I'm not sure it is possible since doctrine/dbal prior v2.12 woudn't work with PHP8 and it's minimum PHP version required is 7.3.